extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -139,9 +177,12 @@ public function getTopAdminsData() {
 			'topAdminsHeaderText' => $this->msg( 'communitypage-admins' )->text(),
 			'otherAdmins' => $this->msg( 'communitypage-other-admins' )->text(),
 			'noAdminText' => $this->msg( 'communitypage-no-admins' )->text(),
-			'noAdminContactText' => $this->msg( 'communitypage-no-admins-contact' )->text(),
+			'noAdminContactText' => $this->msg( 'communitypage-no-admins-contact' )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "noAdminContactText" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "communitypage-no-admins-contact" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)